src/OrleansRuntime/Messaging/IncomingMessageAcceptor.cs

```diff
@@ -363,8 +366,7 @@ private void ProcessAccept(SocketAsyncEventArgs e, bool completedSynchronously)
             }
             catch (Exception ex)
             {
-                var logger = ima != null ? ima.Log : LogManager.GetLogger("IncomingMessageAcceptor", LoggerType.Runtime);
-                logger.Error(ErrorCode.Messaging_IMA_ExceptionAccepting, "Unexpected exception in IncomingMessageAccepter.AcceptCallback", ex);
+                Log.Error(ErrorCode.Messaging_IMA_ExceptionAccepting, "Unexpected exception in IncomingMessageAccepter.AcceptCallback", ex);
```

Comment:
I don't fully understand why they had that logic before, maybe because it was a static method long time ago? Regardless, the check for `ima != null` is all over the place. Maybe you should just in case preserve it closer to what it was before:
```
var logger = ima?.Log ?? this.Log;
```

Reply:
sure. that's better